comp/core/tagger/taggerimpl/collectors/workloadmeta_extract.go

```diff
@@ -72,6 +72,9 @@ var (
 	}
 
 	lowCardOrchestratorEnvKeys = map[string]string{
+		"DD_GIT_COMMIT_SHA":     tags.GitCommitSha,
```

Comment:
Any test for that?

Reply:
Sure!
Here is the corresponding test: https://github.com/DataDog/datadog-agent/pull/27279/files#diff-0ef59cd615f8a571da040730171ca3cd526cd7cad473989015888c5748bf0270R1480